packages/cf-component-box/example/basic/component.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Box } from 'cf-component-box';
+
+const BoxComponent = () => (
+  <Box>
+    <Box margin="1rem" backgroundColor="thistle">
+      This is a box with a margin!
+    </Box>
+
+    <Box padding="1rem" backgroundColor="blanchedalmond">
+      This is a box with padding!
+    </Box>
+
+    <Box
+      padding="1rem"
+      marginTop="1rem"
+      borderTopColor="black"
+      borderTopWidth="1"
+      borderTopStyle="solid"
+
+      borderBottomColor="blue"
+      borderBottomWidth="1"
+      borderBottomStyle="groove"
+
+      borderLeftColor="teal"
+      borderLeftWidth="2"
+      borderLeftStyle="dashed"
+
+      borderRightColor="purple"
+      borderRightWidth="2"
+      borderRightStyle="dotted"
+    >
+      All the borders!
+    </Box>
+
+    <Box display="inline-block" margin="1rem" width="400" backgroundColor="#3dd">
+      half box
+    </Box>
+
+    <Box display="inline-block" margin="1rem" width="400" backgroundColor="#3dd">
+      half box
+    </Box>
+
+    <Box
+      margin="1rem"
```

Comment:
Idea: Should we automatically convert pixels to rems here if used. I use this handy helper https://polished.js.org/docs/#rem. Is there a valid use for pixels nowadays?

```js
<Box margin={15} />
```

would be converted to

```js
<div margin="1rem">
```

in our case where fontSize is 15.